_posts/2020-10-09-utah-sgid-statewide-roads-data-layer-updates-10092020.md

```diff
@@ -0,0 +1,35 @@
+---
+title: Utah SGID Statewide Roads Data Layer Updates 10/09/2020
+author:
+  display_name: Greg Bunce
+  email: gbunce@utah.gov
+date: 2020-10-09 10:30:00
+categories:
+  - Featured
+tags:
+  - roads
+---
+
+AGRC recently completed updates to the SGID Roads feature class. Please visit our [Roads and Highway System]({% link data/transportation/roads-system/index.html %}) data page where you will find information about the Roads data model, as well as direct download links to the SGID Roads data in shapefile and geodatabase format.
+
+These road updates are also reflected in [AGRC's address locators]({% link data/address-geocoders-locators/index.html %}) and the [Utah Next-Generation 911 GIS Database]({% link data/911/index.html %}).
+
+The following are highlights from this month's update.
+
+#### County Updates
+
+New roads were added and road names and address ranges were updated for the following counties:
+
+- **Box Elder County:** Obtained roads data on 09/04/2020. Previous update was on 07/28/2020.
+- **Carbon County:** Obtained roads data on 09/08/2020. Previous update was on 05/20/2020.
+- **Davis County:** Obtained roads data on 09/30/2020. Previous update was on 09/04/2020.
+- **Millard County:** Obtained roads data on 09/08/2020. Previous update was on 08/31/2016.
+- **Salt Lake County:** Obtained roads data on 09/23/2020. Previous update was on 07/28/2020.
+- **Utah County:** Obtained roads data on 09/02/2020. Previous update was on 07/28/2020.
+- **Wasatch County:** Obtained roads data on 09/12/2020. Previous update was on 05/20/2020.
+- **Washington County:** Obtained roads data on 09/30/2020. Previous update was on 09/04/2020.
+- **Weber County:** Obtained roads data on 09/31/2020. Previous update was on 09/04/2020.
+
+#### UDOT Route System
+
+- Read about the [changes coming to the LRS](https://drive.google.com/file/d/1t03AWYHeqTjssTSKnzn60ygdIB-nqEAF/view).

```

Comment:
**question**: Does this doc need to be shared publicly?

Reply:
good catch.  it should be public now.  thanks! 
![image](https://user-images.githubusercontent.com/16906316/95614659-17d70500-0a24-11eb-8199-df8330403330.png)